src/patch/xmlhttprequest.js

```diff
@@ -207,7 +208,7 @@ const Xhook = function() {
 
   // initialise 'withCredentials' on facade xhr in browsers with it
   // or if explicitly told to do so
-  if ("withCredentials" in xhr || xhook.addWithCredentials) {
```

Comment:
I don't see anything about `addWithCredentials` in the README.md, and this property is not part of xhr. Can we remove this condition?

Reply:
I think I added that from some bug I had about 10 years ago which I’ve now forgotten lol

delete it 👍

Reply:
Probably for like IE7 or something 

I think there’s likely more code we can delete because we don’t need to support really old browsers

Reply:
Ok, i get it.